### ISSUE TITLE
Make it possible to override or remove the Crown logotype from the header

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-frontend#1639]
As per https://github.com/alphagov/govuk-design-system-backlog/issues/97#issuecomment-552755688:

> Would it be possible in a future update the ability to change the SVG of the crown? At CH we don't use the crown for our services and there is no simple way to swap it out using Nunjucks

**Comment by kellylee-gds** [comment]
Making the change itself is easy. Understanding how we want teams to use the header and in which circumstances is more complex.

**Comment by DevRCRun** [comment]
As a local authority, consuming the npm module, we're at this point (needing to remove / replace icons and fonts) before we can start testing our first service internally. Looking at macro-options.json for the header component I don't immediately spot an option that would let us change the embedded govuk-header__logotype-crown svg via a param. 

Is there something I'm missing or are we best forking the header component and making the changes we need in our own project?

**Comment by nickcolley** [comment]
@DevRCRun yes you're correct, for now you can't do that with the Nunjucks macro and will need to use HTML directly instead.

**Comment by Dlozitskiy** [comment]
I'm developing a Flask app.

I'm using `govukHeader` and modifying the package directly at `./node_modules/govuk-frontend/dist/govuk/components/header/template.njk` replacing logo SVG, however the changes are not getting reflected after flask restart.

Any tips on what am I missing to get GOV.UK replaced with another logo? cc @NickColley

**Comment by romaricpascal** [comment]
@Dlozitskiy Is your Flask app loading our Nunjucks template directly? I would have thought Python apps would rely on Jinja templates (like [this port of GOV.UK Frontend](https://github.com/LandRegistry/govuk-frontend-jinja)), as Nunjucks is a templating engine for JavaScript (although its syntax is taken from Jinja). If you're using `govuk-frontend-jinja` for your templates, it would be [this template you need to edit](https://github.com/LandRegistry/govuk-frontend-jinja/blob/main/govuk_frontend_jinja/templates/components/header/macro.html) instead of the one in `node_modules/govuk-frontend`.

**Comment by Dlozitskiy** [comment]
Thank you @romaricpascal, that was indeed the case, had to modify `/opt/venv/lib/python3.11/site-packages/govuk_frontend_jinja/templates/components/header/macro.html`

**Comment by matthew-shaw** [comment]
+1 support for this feature request, it would be very helpful for internal services that use govuk-frontend, but de-brand it, or apply other departmental branding. At HMLR we maintain a separate package for internal components from our design system that are 99% the same as the upstream govuk originals, but with things like the logotype, crown, crest etc removed, or replaced with our own branding (as per [the guidance](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk#if-your-service-isnt-on-govuk)). It would be great to just use govuk as-is and be able to configure or disable the logo through a macro parameter/option. Or perhaps a SASS variable, similar to overriding the font and brand colour:
```sass
// Override the default GOV.UK Frontend font stack
$govuk-font-family: "Arial", sans-serif;

// Override the default GOV.UK Frontend brand colour
$govuk-brand-colour: #789904;
```
See https://github.com/matthew-shaw/hmlr-webpack for example. Really appreciate your consideration for other government departments using and re-theming govuk internally!

(Update: Same for https://github.com/alphagov/govuk-frontend/issues/5952)

**Comment by owenatgov** [comment]
Daring to call this closed with the release of the Generic header

**Comment by NickColley** [comment]
See #5952 for footer related unbranding
